sda-svc/templates/doa-deploy.yaml

```diff
@@ -50,6 +50,10 @@ spec:
 {{- if .Values.global.extraSecurityContext }}
 {{- toYaml .Values.global.extraSecurityContext | nindent 10 -}}
 {{- end }}
+      {{- if .Values.global.vaultSecrets }}
+        command: ["/bin/sh"]
+        args: ["-c", "source {{ include "secretsPath" . }}/{{.Values.global.confFile}} && docker-entrypoint.sh && java -jar /localega-doa.jar"]
```

Comment:
I'm not sure I manage to keep track, so just to check - is this really sourcable?

Reply:
It all depends on how the file created by Vault looks like, but for DOA to start the file should look like this:

```
export BROKER_USER=<username>
export BROKER_PASSWORD=<password>

```

Reply:
Made it a bit clearer for the DOA case